src/browser/modules/Stream/CypherFrame/VisualizationView.tsx

```diff
@@ -24,23 +24,26 @@ import { withBus } from 'react-suber'
 import { Action, Dispatch } from 'redux'
 import { Bus } from 'suber'
 
-import { GraphModel } from 'graph-visualization'
+import { GraphModel, Explorer } from 'graph-visualization'
 
-import Explorer from '../../D3Visualization/components/Explorer'
 import { StyledVisContainer } from './VisualizationView.styled'
 import { resultHasTruncatedFields } from 'browser/modules/Stream/CypherFrame/helpers'
 import bolt from 'services/bolt/bolt'
-import {
-  BasicNode,
-  BasicNodesAndRels,
-  BasicRelationship
-} from 'services/bolt/boltMappings'
+import { BasicNode, BasicNodesAndRels, BasicRelationship } from 'common'
 import { NEO4J_BROWSER_USER_ACTION_QUERY } from 'services/bolt/txMetadata'
-import { deepEquals } from 'services/utils'
+import { deepEquals } from 'common'
 import { GlobalState } from 'shared/globalState'
 import { CYPHER_REQUEST } from 'shared/modules/cypher/cypherDuck'
 import * as grassActions from 'shared/modules/grass/grassDuck'
-import { getMaxFieldItems } from 'shared/modules/settings/settingsDuck'
+import {
+  getMaxFieldItems,
+  shouldShowWheelZoomInfo
+} from 'shared/modules/settings/settingsDuck'
+import {
+  getNodePropertiesExpandedByDefault,
+  setNodePropertiesExpandedByDefault
+} from 'shared/modules/frames/framesDuck'
+import * as actions from 'project-root/src/shared/modules/settings/settingsDuck'
```

Comment:
We already import `getMaxFieldItems` and `shouldShowWheelZoomInfo` from settingsDuck, perhaps we could import `update` from there as well. We could rename it as well.

```ts
import {
  getMaxFieldItems,
  shouldShowWheelZoomInfo,
  update as updateSettings
} from 'shared/modules/settings/settingsDuck'

```

 rather than import everything? wdyt?

Reply:
yes I agree, thanks!